src/nvim/api/ui_events.in.h

```diff
@@ -47,19 +47,22 @@ void visual_bell(void)
 void flush(void)
   FUNC_API_SINCE(3) FUNC_API_REMOTE_IMPL;
 void update_fg(Integer fg)
-  FUNC_API_SINCE(3);
+  FUNC_API_SINCE(3) FUNC_API_BRIDGE_IMPL;
 void update_bg(Integer bg)
-  FUNC_API_SINCE(3);
+  FUNC_API_SINCE(3) FUNC_API_BRIDGE_IMPL;
 void update_sp(Integer sp)
-  FUNC_API_SINCE(3);
+  FUNC_API_SINCE(3) FUNC_API_BRIDGE_IMPL;
+void default_colors_set(Integer rgb_fg, Integer rgb_bg, Integer rgb_sp,
+                        Integer cterm_fg, Integer cterm_bg)
```

Comment:
Maybe it's worth adding a stub for `cterm_sp` now (IIUC it's needed for https://github.com/neovim/neovim/issues/7479 )

Reply:
I would expect any terminal that supports sp would also support truecolors, and would be used in this mode when sp is wanted. (not to mention, vim doesn't have `ctermsp=` at all) The relevant change for #7479 would rather be to store `rgb_sp` on the tui.c side.